### ISSUE TITLE
rtsp with H.264 support export format 

### ISSUE THREAD

**Issue by brusezot** [iamscottxu/obs-rtspserver#130]
Thank you for this develop, this add-on, do you have an idea if it's possible to develop and include the h.264 export format ?

**Comment by iamscottxu** [comment]
Sorry. I don't understand. Please describe your problem in detail.

**Comment by brusezot** [comment]
Hi @iamscottxu, I need to run obs RTSP server with Dartfish Software, Do you this software ?

**Comment by iamscottxu** [comment]
The encoder used by the obs-rtspserver plugin depends on the live broadcast settings of OBS. If your dartfish and OBS are not installed on the same computer, please pay attention to the firewall settings of the computer where OBS is installed.

<img width="738" alt="image" src="https://user-images.githubusercontent.com/22959917/225913966-45863de9-f5d3-484e-ae31-4eb5d2500c59.png">

**Comment by brusezot** [comment]
Yes, I understand, but, I just want to verify with you that it works properly with my software. Is that possible? If so, can I send you the Dartfish software to test ?

**Comment by iamscottxu** [comment]
> Yes, I understand, but, I just want to verify with you that it works properly with my software. Is that possible? If so, can I send you the Dartfish software to test ?
> 
> Give me your email address to receive a Dartfish license for testing

[scottxu@scottxublog.com](mailto:scottxu@scottxublog.com)

**Comment by iamscottxu** [comment]
<img width="1420" alt="image" src="https://user-images.githubusercontent.com/22959917/225923782-c167ed35-8082-4b15-b12b-d3a096d643f0.png">
<img width="417" alt="image" src="https://user-images.githubusercontent.com/22959917/225924030-73075083-962f-4536-9522-bf7ebbcad4c4.png">

**Comment by brusezot** [comment]
I want just your opinion about this service is stable with my software because actually we recommand fo our sales to used a encoder H.264 box (Blackmagic / Avermedia / Blankam), and me i purpose to use only software OBS.

**Comment by iamscottxu** [comment]
When I using VLC to play, the delay is about two seconds. But when I use Dartfish, the delay is about six seconds, and the video and audio are out of sync. If turn off recording, the delay is about four seconds.

**Comment by brusezot** [comment]
Ok,

Thank you. What do you think about this delay ? Why ? How is possible to resolved this problem ?
